ocs_ci/helpers/cephx_keyrotation_helper.py

```
@@ -0,0 +1,4515 @@
+        items = resources.get("items", [])
+        if not items and resources.get("metadata"):
+            items = [resources]
+
+        entities = []
+        for item in items:
+            name = item.get("metadata", {}).get("name")
+            if not name:
+                continue
+            for candidate in (f"client.{name}", f"client.ceph-{name}"):
+                if self._auth_entity_exists(candidate, toolbox_pod):
+                    entities.append(candidate)
```

Ruff (0.16.0)

[warning] Use `list.extend` to create a transformed list

(PERF401)